components/gitpod-db/src/typeorm/migration/1666616345054-AddLastModifiedToWorkspaceClusterTable.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+const TABLE_NAME = "d_b_workspace_cluster";
+const COLUMN_NAME = "_lastModified";
+
+export class AddLastModifiedToWorkspaceClusterTable1666616345054 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE ${TABLE_NAME} ADD COLUMN ${COLUMN_NAME} timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6), ALGORITHM=INPLACE, LOCK=NONE `,
```

Comment:
I think you'll also need an index on this. Wanna add here?

Reply:
The table will only contain ~4 rows so I'm not sure it's worth it.

Reply:
Fair, tho I'd still add it for consistency with other tables. Up to you.

Reply:
rebased to add the index